study.md

```diff
@@ -39,7 +60,12 @@ sum = left_operand.to_i + right_operand.to_i
 Write some JavaScript code that duplicates this functionality.
 
 ```javascript
-// your answer here
```

Comment:
We want the Ruby equivalent of this code.  It should be something like the following.

```javascript
const leftOperand = '1';
const rightOperand = '2';

const sum = parseInt(leftOperand) + parseInt(rightOperand);
```